platform/darwin/src/NSExpression+MGLAdditions.mm

```diff
@@ -792,6 +792,9 @@ + (instancetype)expressionWithMGLJSONObject:(id)object {
             } else if ([curveType isEqualToString:@"cubic-bezier"]) {
                 curveParameters = @[@"literal", [interpolationOptions subarrayWithRange:NSMakeRange(1, 4)]];
             }
+            else {
+                curveParameters = [NSNull null];
```

Comment:
This makes sense because NSExpression itself allows `[NSExpression expressionForConstantValue:nil]` and `[NSExpression expressionWithFormat:@"NIL"]`, so we usually tell developers to write `NIL` as the parameters argument. This bridges the mismatch between NSExpression and JSON, which would represent the same value as NSNull.